public/views/listProblems/listProblems.html

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+    <!--
+    <div class="page-header">
```

Comment:
Are you planing in uncommenting this code or you forgot to remove it? We would prefer to avoid to not have commented out code like this unless it's there because something else needs to be done before you uncomment it.